org.eclipse.lsp4e/src/org/eclipse/lsp4e/LSPProgressManager.java

```diff
@@ -0,0 +1,158 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Avaloq Evolution AG.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Rubén Porras Campo (Avaloq Evolution AG) - initial implementation
+ *******************************************************************************/
+package org.eclipse.lsp4e;

```

Comment:
Would it be possible to have this in a `org.eclipse.lsp4e.progress` package?

Reply:
done